tests/test_resource.py

```diff
@@ -61,6 +61,15 @@ def test_create_resource_type(self):
         resource_types = ResourceType.get_types()
         self.assertEqual([resource_type, resource_type2], resource_types)
 
+        # create an additional resource type
+        resource_type2 = 'test_type2'
```

Comment:
This should fix the issue that's causing tests to fail:

```suggestion
        resource_type2 = 'test_type3'
```

Reply:
(although variables should be renamed to `resource_type3` and `test_type3` to reduce confusion...)